test/unit/helper/WebDriverIO_test.js

```diff
@@ -442,6 +442,12 @@ describe('WebDriverIO', function () {
         .then(() => wd.grabAttribute({css: 'form'}, 'method'))
         .then((val) => assert.equal(val, "get"));
     });
+
+    it('should grab css from element', () => {
+      return wd.amOnPage('/')
+        .then(() => wd.grabCssFrom('#area1', 'prop'))
```

Comment:
you are not getting a CSS property here, but a value of an attribute. That's what method `grabAttribute` does.